components/widgets/modals/full_screen_modal.tsx

```diff
@@ -32,6 +34,22 @@ export default class FullScreenModal extends React.Component<Props> {
         }
     }
 
+    private onBackKeyDown = (e: KeyboardEvent) => {
+        const code = e.which;
+        if ((code === 13) || (code === 32)) {
```

Comment:
Can we use the key codes from `constants.jsx` to increase readability?

Reply:
I going to replace it with a button, so it won't be needed.